dataland-frontend/src/components/pages/DataPrivacy.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="pb-8">
+  <div class="pb-8 d-content">
```

Comment:
Why was this added? Is it to remove the white space at the bottom of the page?

Reply:
Das weiß ich leider nicht.  Aber ich habe das eben mal im Browser betrachtet, und es scheint einfach wirklich nur den whitespace zu ergänzen.
Besonders viel Logik scheint da auch nicht drin zu stecken:

![image](https://github.com/d-fine/Dataland/assets/99255296/320300d3-9275-4c83-9553-047153c07761)


Ist das trotzdem ok für dich, oder soll ich es löschen?

Reply:
Ne das sieht finde ich besser aus. Ich würde das drin lassen und hier resolven